loops/_voice_tracker.py

```
@@ -0,0 +1,51 @@
+    channel_members = after.channel.members if after.channel else []
+    active_members = [member for member in channel_members if not (member.voice.self_mute or member.voice.self_deaf)]
+
+    if len(active_members) < 2:
+        for member in channel_members:
+            removeVoiceUser(member)
+    else:
+        updateVoiceUsers(active_members)
+
+
+def updateVoiceUsers(active_members: list[discord.Member]) -> None:
+    global voiceUsers
```

Flake8 (7.3.0)

[error] `global voiceUsers` is unused: name is never assigned in scope

(F824)